Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.h

```diff
@@ -74,9 +76,18 @@ namespace AzToolsFramework::Prefab
         void OnPrefabTemplateDirtyFlagUpdated(TemplateId templateId, bool status) override;
         
     private:
-        PrefabFocusOperationResult FocusOnPrefabInstance(InstanceOptionalReference focusedInstance);
+        enum FocusChangeBehavior
```

Comment:
update: We should make this `enum class`

Reply:
Bother. Thanks, fixed.